eclcli/common/utils.py

```diff
@@ -347,6 +347,20 @@ def validate_ipv4(text):
         raise exceptions.CommandError(msg % text)
     return text
 
+def validate_int_range(text, min_val, max_val):
+    try:
+        if not isinstance(text, int):
+            int_text = int(text, 0)
```

Comment:
It is enough to support decimal numbers
```suggestion
            int_text = int(text)
```